# HG changeset patch
# User BelovaMM
# Parent  467764f561b740c8e23f44b73668ea8ad550721b
refs #DEVBO-753 Проверка возраста MAX ID. Отладка. #BTS-2580
BTS-2580 На ГКС не передается параметр NeedUserSimpleAgeConfirm

https://msk-exp-jira.expertek.local/browse/DEVBO-753
https://msk-exp-jira.expertek.local/browse/BTS-2580

diff --git a/str/send-all-work-gismt.p b/str/send-all-work-gismt.p
--- a/str/send-all-work-gismt.p
+++ b/str/send-all-work-gismt.p
@@ -340,7 +340,7 @@ procedure put-xml-data-ui:
     
     iSAXWriter:start-element("Param") .
     iSAXWriter:insert-attribute("ctrl", "ADD").
-    iSAXWriter:insert-attribute("group", "UISettings").
+    iSAXWriter:insert-attribute("group", "UiSettings").
     iSAXWriter:insert-attribute("key", p-prop-code).
     iSAXWriter:write-data-element("ParamValue" , p-value ) .
     iSAXWriter:write-data-element("ParamDesc" , p-discr).
